### DISCUSSION TITLE
Keyball39の12番キーのスイッチソケットのランド剥がれ

### DISCUSSION THREAD

**Discussion by JIUJIE7T** [Yowkees/keyball#678] [Q&A] [answered]
お世話になっております。
Keyball39を組み立てましたが、いくつかのキーが反応しませんでした。
はんだ付けをし直してみて、一部のキーが治りました。
12番キー（左手の「S」にあたるキー）をスイッチを付けた最初の数秒間だけ反応があり、その後反応しなくなります。

再度はんだ付けしなおすのき、パッドが剥がれました。
直す方法がございますでしょうか。

ご確認いただければ幸いです。
![ebdc31103d74dc820a35d6f28185430](https://github.com/user-attachments/assets/9d14e853-8f77-48ea-8176-6f24400fc7a6)
![bc0d3b07dd4e43235015cc6c6a87f22](https://github.com/user-attachments/assets/cbd48099-aa1b-475b-820e-d9b840230245)

![6a45064ef1eb6096f11a2a178a5ed4a](https://github.com/user-attachments/assets/b2f9516a-90cc-4b39-a739-3c373447b7b6)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

下記画像の緑丸で示したダイオードの端子がテンプラしているように見えます。このダイオードが問題のキーに対応していますので、ここを直すと復活できたかも知れません。
剥がれてしまったパッドの復活方法ですが、緑丸どうしをエナメル線等で接続していただくと復活できます。
ただ、キーソケットの固定力が弱くなったので、両面テープ等で固定することをお勧めします。
![KB39L_keyfix](https://github.com/user-attachments/assets/9cfc6af8-6884-45fe-a5c0-fec21062cbf6)

もし、すべての半田付けをやり直したい場合には、ミドル基板と半田付け部品セットも、Keyballご購入者様限定で、比較的安価に販売できますので、白銀ラボウェブショップよりお問い合わせいただければ幸いです。
https://shirogane-lab.net/inquiry

以上、ご検討よろしくお願いいたします。

**Reply by JIUJIE7T** [reply]
ご確認ありがとうございます。
資材に調達するに時間がかかりました。返信が遅くなりました。

示したポイントをエナメル線で繋いで、キーボードが無事に動くようになりました。
ありがとうございます。

**Comment by Yowkees** [comment]
無事に動くようになったようで良かったです！安心しました。
ぜひ末永くKeyball39をご愛用いただければ幸いです。

**Comment by JIUJIE7T** [comment]
@Yowkees 
すみません、同じスレッドで恐縮ですが、usbケーブルが左右のPro Microに挿す場合、マウスカーソルが異なる速度（DPI？）で動く事象が発生しています。
左に接続する際マウスの動きがとても遅く、逆に右に挿す場合普通にスムーズに動きます。
Remapを確認しましたが特にDPIを制御するマッピングがありませんでした。
こちらはKeyballハード面で制御しているのでしょうか。
今は左側に接続する際のトラックボールを通常の速度にもどしたく、質問させていただきました。
ご確認いただけますと幸いです。
よろしくお願いいたします。
